cdn-uploader.js

```diff
@@ -0,0 +1,45 @@
+const ftp = require("basic-ftp")
+
+const config = {
+host: process.env.AKAMAI_CDN_HOSTNAME,
+user: process.env.AKAMAI_CDN_USERNAME,
+password: process.env.AKAMAI_CDN_PASSWORD,
+cpCode: process.env.AKAMAI_CDN_CPCODE,
+port: 21,
+basePath: "./builds"
+}
+
+function UploadFilesFromPath(sources) {
+	process.chdir(config.basePath);
+	console.log("[+] Connecting to CDN...");
+	for (let i = 0; i <= sources.length-1; i++) { 
+		async function Upload() {
+			try {
+				const client = new ftp.Client();
+				await client.connect(config.host, config.port);
+				await client.login(config.user, config.password);
+				await client.useDefaultSettings();
+				console.log(`[+] Starting to upload the content of "${sources[i]}".`);
+				await client.ensureDir(`/${config.cpCode}/${sources[i]}`);
+				await client.clearWorkingDir();
+				await client.uploadDir(sources[i]);
+				client.close()
+				console.log(`[+] Folder "${sources[i]}" uploaded.`);
+			}
+			catch(err) {
+				console.log(err)
+			}
+		}
+		Upload();
+	}
+}
+
+var folders = [];
+process.argv.forEach(function (val, index, array) {
+	folders.push(array[index]);
+});
+folders.shift();
+folders.shift();
+config.basePath=folders[0];
+folders.shift();
```

Comment:
What is the meaning of these shifts? 

Do you think that it could be possible having a more general approach?

For example:

```js
var folders = process.argv.skip(3).map(x => ...
var config.basePath = process.argv[2];
...
```